fwdpy11/discrete_demography.py

```diff
@@ -535,3 +525,16 @@ def _timed_events(self):
             self.set_migration_rates,
         ]:
             yield i
+
+
+def from_demes(
```

Comment:
Let's get the "versionadded" bit here, too, as this is the user-facing function.